app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes
+  has_many :comments
+
+  # returns the 3 most recent posts for a given user
+  def self.three_most_recent(author_id)
+    Post.where(author_id:).order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] [OPTIONAL] I would suggest that you take full advantage of the `has_many` relationship between the user and their posts like this 👇 
```ruby
def three_most_recent
  posts.order('created_at Desc').limit(3)
end
```